build/xwalk.gni

```diff
@@ -7,4 +7,13 @@ declare_args() {
   # Whether to build and use Crosswalk's internal extensions (device
   # capabilities, sysapps etc).
   disable_bundled_extensions = false
+
+  # Android: whether the integrity of the Crosswalk runtime library should
+  # be verified when Crosswalk is run in shared mode.
+  verify_xwalk_apk = 0
```

Comment:
It is on purpose set as 0,  see line 124 of `//xwalk/tools/reflection_generator/reflection_generator.py`